### PR TITLE
408 deployment enhancement

### PR DESCRIPTION
**Pull request by lukas0820**
- every service is now available from port 80 / 443
- mapping:
  - server: /media, /api, /web
  - app: /webapp
  - web: /admin
 - database is only available inside docker compose
 - added new page on / for service selection
 - TLS is only handled by nginx reverse proxy

### CODE REVIEW

**Comment by lukas0820** [comment]
closes #408

**Review by YIllmann — APPROVED** [review]

**Review by n00on — CHANGES_REQUESTED** [review]
I like the changes really much, however the nginx service would always exit at first for me:
Screenshot From 2024-09-22 23-37-06

On restart it  worked.

I'd suggest adding to nginx service in docker compose:
```yaml
    depends_on:
      - server
      - web
      - app
```

File: index.html [hunk 1]
```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="de">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MANV Simulations-App</title>
```
Comment (n00on) [inline index.html:6]: add the line: 
```html
<link rel="icon" type="image/x-icon" href="/webapp/favicon.png">
```
here to add the favicon from the flutter web app

Comment (lukas0820) [reply index.html:6]: Done

**Pull request merged**